### DISCUSSION TITLE
Termux file empty

### DISCUSSION THREAD

**Discussion by chicoenu** [termux/termux-app#2231] [Q&A]
So I have Samsung android 11 and i tried to check the termux folder in my device, since android 11 denies access to /Data/ folder i tried "CX File Explorer" it shows /Data/ folder with all the files except the /Termux/ folder it says (Empty) i saved couple files in Termux and now i can't see them.. and i don't think changing the directory works cause well i can't get back the files i need.. any help would be so much appreciated!.

**Comment by agnostic-apollo** [comment]
Check https://wiki.termux.com/wiki/Internal_and_external_storage and https://github.com/termux/termux-packages/wiki/Termux-file-system-layout. You likely stored files in `$HOME` (`/data/data/com.termux/files/home`), not `/sdcard/Android/data/com.termux`.

**Reply by chicoenu** [reply]
The problem is that when i click on termux folder.. it's empty there's nothing in it

**Reply by agnostic-apollo** [reply]
Run `ls -a ~` in termux. If there are no files shown, then you didn't create it. If shown, then CX file explorer is broken and use another app. Note that hidden files starting with `.` will not be shown in Termux SAF in old versions like the one on playstore, use F-Droid (latest) version for fix.
